### PR TITLE
AG 0.8.3

### PR DESCRIPTION
**Pull request by suzhoum**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by suzhoum** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ requirements:
   run:
     - python >=3.8,<3.11
     - numpy >=1.21,<1.27
-    - pandas >=1.4.1,<2.2
+    - pandas >=1.4.1,<1.6
```
Comment (Innixma) [inline recipe/meta.yaml:28]: Why did 0.8.2 have <2.2 for pandas? The version range was <1.6 in the 0.8.2 release: https://github.com/autogluon/autogluon/blob/v0.8.2/core/src/autogluon/core/_setup_utils.py#L22

**Review by Innixma — APPROVED** [review]
LGTM if the comment is addressed

File: recipe/meta.yaml (hunk 1 above)
Comment (suzhoum) [reply recipe/meta.yaml:28]: Yes I noticed that too, so I've updated it to match with the release.

Comment (Innixma) [reply recipe/meta.yaml:28]: Can you explain why it was 2.2 to begin with?

Comment (Innixma) [reply recipe/meta.yaml:28]: It was probably 2.2 for a reason.

Comment (suzhoum) [reply recipe/meta.yaml:28]: @tonyhu, do you have any insights?

Comment (tonyhoo) [reply recipe/meta.yaml:28]: This is a requirement from sagemaker-distribution. See here codes at [here](https://github.com/aws/sagemaker-distribution/blob/c9873bb29c0d1ccb2fbdd20391fb509ddc70602b/build_artifacts/v1/v1.0/v1.0.0/cpu.env.in#L8)

Comment (tonyhoo) [reply recipe/meta.yaml:28]: We need to update this to 2.2

Comment (suzhoum) [reply recipe/meta.yaml:28]: Thanks, will update this.

Comment (Innixma) [reply recipe/meta.yaml]: Thanks for verifying and adjusting

**Pull request merged**